nifi-nar-bundles/nifi-asn1-bundle/nifi-asn1-services/src/main/java/org/apache/nifi/jasn1/JASN1Reader.java

```diff
@@ -134,17 +135,32 @@ public class JASN1Reader extends AbstractConfigurableComponent implements Record
         .required(false)
         .build();
 
+    private static final PropertyDescriptor PREPROCESS_OUTPUT_DIRECTORY = new PropertyDescriptor.Builder()
```

Comment:
I think the misconception here is that we only need the output directory so that we can look at the resulting asn files. That is not true, it is also required for NiFi itself.
With that in my I think creating 2 separate properties - neither of which making sense without the other - would only clutter the configuration of the processor uncessarily.